_usage/command-line.md

```diff
@@ -208,5 +206,5 @@ INSERT INTO some_table VALUES (1);
 GO
 ```
 
```

Comment:
The rest of this section doesn't make sense to me:
Do we want the users to set the value of sql_dialect with psql?
Why are they inserting values into some_table right after that - it's a strange example?
If they do set the value of sql_dialect, do they need to restart the server?
What's wrong with ordinary users (line 210) using psql as long as they connect to the PG port?